src/player.js

```diff
@@ -356,10 +356,11 @@ let Player = {
                 const exp = loadedPoke[1];
                 const shiny = (loadedPoke[2] === true);
                 const caughtAt = loadedPoke[3];
+                const prestigeLevel = loadedPoke[4] || 0;
```

Comment:
The save loading code should probably be simplified a bit so we don't have to change everything in 5 different locations in the future, that could get tiresome (and is somewhat prone to breaking)